opentelemetry-etw-metrics/examples/basic-metrics.rs

```diff
@@ -0,0 +1,100 @@
+use opentelemetry::{
+    metrics::{MeterProvider as _, Unit},
+    KeyValue,
+};
+use opentelemetry_etw_metrics::MetricsExporter;
+use opentelemetry_sdk::{
+    metrics::{PeriodicReader, SdkMeterProvider},
+    runtime, Resource,
+};
+use std::{thread, time::Duration};
+
+const SERVICE_NAME: &str = "hello-geneva-user-events";
+const METRICS_ACCOUNT: &str = "your-geneva-metrics-account";
+const METRICS_NAMESPACE: &str = "your-geneva-metrics-namespace";
+
+fn setup_meter_provider() -> SdkMeterProvider {
+    let exporter = MetricsExporter::new();
+    let reader = PeriodicReader::builder(exporter, runtime::Tokio)
```

Comment:
let reader = PeriodicReader::builder(exporter, runtime::Tokio).build();

default interval is good

Reply:
Updated to just use the default interval